Cohen-Koen.md

```diff
@@ -0,0 +1,32 @@
+﻿---
+name: "@Cohen-Koen"
+project: "Screens for Cougar MFD"
+---
+
+#  Screens for Cougar MFD
+
+##  Summary
+
+I am going to build and integrate screens with the Thrustmaster Cougar MFD, this will allow for a realistic look on MFD, I wish to make an affordable option for anyone who works with flight simulators like me. There are alternatives however I wish to make them well integrated and something that people can use as a guide for their own projects
+
+##  Plan
+
+Once my parts arrive, I will build a mount for my MFD with the screen.
+Second, I will setup a software or etc. to allow the user sim of choice to operate with the screen.
+Throughout this project I will show how this is done with good photos and documentation on Gitbook and I will share the process to help the flight simulator community
```

Comment:
You should document this in your [Hack Club Scrapbook](https://scrapbook.hackclub.com). Join the `#scrapbook` channel on the Hack Club Slack to get started!

Reply:
Sounds great! I will learn how to use that, its my first time doing a scrapbook. All my previous writing has been very formal, a paper on the Riemann Zeta Function has been a piece of my past work.